modules/services/dnsmasq.nix

```diff
@@ -0,0 +1,71 @@
+{ pkgs, lib, nglib, config, ... }:
+with lib; with nglib;
```

Comment:
In the future I plan to remove all or most usages of `with`, especially `with lib/nglib`. Ideally this would be replaced by either explicit calls to `lib.mkDefaultRec` or 
```nix
let
  inherit
    (nglib)
    mkDefaultRec
    ;
in